### ISSUE TITLE
RangeSelector.ts: Cannot read properties of undefined (reading 'options')

### ISSUE THREAD

**Issue by timkendrick** [highcharts/highcharts-react#490]
Stock chart with `rangeSelector` throws an unhandled exception under the following scenario:
 
1. Render with a non-empty `series` list that contains multiple data points, with `rangeSelector` options specified
2. Update the range selection to show a subset of the data points (either via UI controls or via `chart.update()` API)
3. Re-render with an empty `series` list, with `rangeSelector` options specified
4. Attempt to re-render with either an empty or a non-empty `series` list, with `rangeSelector` options specified
 
Reproduction: https://stackblitz.com/edit/react-dfktmn?file=index.js
 
There's also a more minimal reproduction that doesn't rely on changing the range selection:
 
1. Render with a non-empty `series` list containing no data points, and `rangeSelector` options specified
2. Re-render with an empty `series` list, with `rangeSelector` options specified
3. Attempt to re-render with either an empty or a non-empty `series` list, with `rangeSelector` options specified
 
Reproduction: https://stackblitz.com/edit/react-gtwuxq?file=index.js
 
---
 
Analysis:
 
This seems to be due to the following line in `RangeSelector.ts`:
 
https://github.com/highcharts/highcharts/pull/15615/files#diff-ac661cc83486297b5dc44443dde377e95cfe4b0e01f1e4af70553504e32a6ee9
 
```
            if (chart.navigator && chart.navigator.baseSeries[0]) {
                chart.navigator.baseSeries[0].xAxis.options.range = void 0 as any;
            }
```
 
this throws an unhandled exception because `chart.navigator.baseSeries[0].xAxis` is `undefined`

**Comment by ppotaczek** [comment]
Hi @timkendrick,

Thank you for contacting us! 
The issue is related to Highstock, not to the `highcharts-react` integration.
However, the problem has been fixed in Highcharts v11.4.0, so the easiest way to make it work is to update your dependencies.

**Live example:** https://stackblitz.com/edit/react-5taefm

Best regards!

**Comment by timkendrick** [comment]
That's fantastic news, thanks for the speedy response @ppotaczek!
